client/web/compose/src/components/Common/RecordToolbar.vue

```diff
@@ -27,6 +27,29 @@
         </b-button>
       </div>
 
+      <b-button-group class="ml-auto">
+        <b-button
+          v-if="nextPrevNavigationOptions.prev"
+          pill
+          size="lg"
+          variant="outline-primary"
+          class="mr-2"
+          @click="$emit('update-navigation:prev')"
+        >
+          <font-awesome-icon :icon="['fas', 'angle-left']" />
+        </b-button>
+
+        <b-button
+          v-if="nextPrevNavigationOptions.next"
+          size="lg"
+          pill
+          variant="outline-primary"
+          @click="$emit('update-navigation:next')"
```

Comment:
I suggest having a single event, and using value to indicate the prev/next.
For example -1 for prev, 1 for next.